COM-50 Indicator 15

lauravignoli opened the pull request.

lauravignoli committed:
COM-50 Indicator 15

prashantsable1991 commented:
@lauravignoli you really did a great job but unfortunately, the report is not generating while following below test cases:
https://docs.google.com/document/d/1EcAtr25C5EOrmG_gKTZ1Lehg9FhZh_E2xHhUKxjeJSE/edit

Also, some functions are failing while some are getting executed in the terminal. Please refer below screenshot:

![COM-50(1)](https://user-images.githubusercontent.com/47622485/59514376-574a1b80-8eda-11e9-84bd-d3948a8512ba.png)

![COM-50(2)](https://user-images.githubusercontent.com/47622485/59514385-5d3ffc80-8eda-11e9-96a3-73a8f11ec4f7.png)

prashantsable1991 commented:
@lauravignoli Please retest after destroying vagrant.

lauravignoli committed:
COM-50 Update indicators function

Updated patientWasOnARVOrHasPickedUpADrugWithinPeriodPlusOrMinusMonths. Before it was checking the entire period including the reporting months plus and minus months. It was updated to consider the reporting month minus months OR reporting month plus months.
